.gitignore

```diff
@@ -1,4 +1,3 @@
 tests/*.out
 zig-cache/
-zig-out/
```

Comment:
Why this change?

Reply:
What do you mean

Reply:
Why was this line removed from gitignore?

Reply:
Why? Shouldn't it be removed.